modules/ROOT/pages/runtime-monitoring.adoc

```diff
@@ -58,154 +61,39 @@ The following available metrics are *disabled by default* and can be enabled.
- #org.bonitasoft.engine.monitoring.publisher.logging.step=PT1M
-
-These are the *metric publishers* that can be activated. +
-In Community edition, 2 metric publishers are provided:
-
-* JMX (enabled by default), that allows to use any JMX console to monitor your favorite metrics (except JVM metrics,
-as they are already published by the JVM itself by default)
-* Logging (disabled by default), that regularly prints to standard Bonita log file the Bonita-related metrics. Print interval can
-be changed (property `org.bonitasoft.engine.monitoring.logging.step`).
-
-[NOTE]
-====
-
-To change any value, *uncomment the line by removing the # character*, and change the true / false value. +
-Then xref:BonitaBPM_platform_setup.adoc]#update_platform_conf[push your configuration changes] to database:
-
-[source,bash]
-----
-./setup/setup.sh push
-----
-
-Then restart the Tomcat server for the changes to take effect.
-====
-
-=== Activating specific metric indicators
-
-Edit the same file `./setup/platform_conf/current/platform_engine/bonita-platform-community-custom.properties`
-
- ## METRICS = what to publish?
- ##
- ## Note: Bonita-related metrics are automatically published.
- ## They are active by default and cannot be disabled.
- ##
- ## publish technical metrics related to Worker / Connector thread pools:
- #org.bonitasoft.engine.monitoring.metrics.executors.enable=false
- ## publish technical metrics related to HIBERNATE statistics
- ## To activate, simply set property (a few lines above) 'bonita.platform.persistence.generate_statistics=true'
-
-These are the *metrics* (counters) that can be exposed. +
-All configurable metrics are disabled by default and can be enabled separately. +
-They provide information about:
-
-* Worker / Connector thread pools
-* Hibernate statistics
-
-Each of these metrics provides many different counters to finely understand what is going on.
-
-[NOTE]
-====
-
-To change any value, *uncomment the line by removing the # character*, and change the true / false value. +
-Then xref:BonitaBPM_platform_setup.adoc]#update_platform_conf[push your configuration changes] to database:
-
-[source,bash]
-----
-./setup/setup.sh push
-----
-
-Then restart the Tomcat server for the changes to take effect.
-
-====
+== How to retrieve those metrics
 
-== Subscription-only monitoring
+Those *metrics* are published on *publishers*. *Publishers* are responsible for exposing the activated metrics.
+It is a channel on which metrics are published. Provided metric publishers are JMX, Log files, Prometheus.
 
-=== Additional metrics
+=== JMX Publisher
 
-Thanks to additional publisher, additional metrics can be published
+The JMX publisher publishes each metrics using MBeans. All those MBeans are in the domain `metrics`.
 
-These metrics provide information about:
+It can be enable using the property `org.bonitasoft.engine.plugin.monitoring.publisher.jmx.enable` set to `true`.
```

Comment:
⚠️ Remove extra `.plugin`
```suggestion
It can be enable using the property `org.bonitasoft.engine.monitoring.publisher.jmx.enable` set to `true`.
```

Reply:
good catch 👍